### PR TITLE
Adding dockerfile for ETL script

### PR DESCRIPTION
**Pull request by bentrzcinski**
Adding a dockerfile to build a docker image for the ETL script ready to be uploaded to the ECR. Also rearranged the scripts and put some in to folders for tidiness sake. The dockerfile currently references scripts that don't exist yet, namely the 'load.py' and 'etl.py' scripts. But we expect these to be made in the future to make the pipeline fully operational. Names may need to be kept in mind or amended at a future point to ensure the dockerfile works.

### CODE REVIEW

**Review by anitamegarry — DISMISSED** [review]
Looks good to me!

**Review by jiuliangut — APPROVED** [review]
Looks good - nice descriptive readme :)

File: pipeline/dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+RUN apt-get update && apt-get install -y \
+    freetds-bin \
+    freetds-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
```
Comment (anitamegarry) [inline pipeline/dockerfile:13]: You need to copy the .env file so that you can connect to the database.

**Review by anitamegarry — APPROVED** [review]
Nice!!

**Pull request merged**